plugins/Login/templates/invitation.twig

```diff
@@ -1,48 +1,57 @@
+{% set isCenteredLogo = false %}
+{% set showImageDiv = true %}
+{% set imageToShow = 'plugins/Login/images/01_Create password.png' %}
+{% set colClassOverride = 'col l4 offset-l2 m12' %}
+{% set bodyClass = 'onBoarding' %}
 {% extends '@Login/loginLayout.twig' %}
 
 {% block loginContent %}
-    <div class="contentForm invitationForm">
-        <div class="card">
-            <div class="card-content">
-                <div class="card-title">
-                    {{ "Login_InvitationTitle"|translate }}
-                </div>
+    <div class="contentForm invitationForm" id="invitationForm">
+        <p class="uppercase header-p m-top">{{ 'General_CreatePassword'|translate }}</p>
+        <p>&nbsp;</p>
+        <p class="header-p">{{ 'Login_CreatePasswordDescription'|translate }}</p>
+        <p>&nbsp;</p>
```

Comment:
@sgiehl I don't agree that having classes like `uppercase` or `m-top` is bad as frameworks like Bootstrap / Materialize has this kind of classes to make styling easier and reusable.
If you strongly think we should not use it, I am happy to change it :smile: 

I have removed empty p and used margins instead

Reply:
The markup is not ideal but is better now than what it was first time around.

There are different methodologies around using CSS classes and naming things. For now I think using the boostrap/materialise/tailwind-like approach of utility classes is ok and I thought somewhat consistent with what we already had, being a bit friendlier for developers leaning towards the back-end side of things (as opposed to pure front-end coders). And since HTML can take a lot, we don't focus on a correct semantic and a11y-compliant structure as much as we should.

On the other hand, we might be using a bit more descriptive classnames here as we are already mixing in things like `.footerLinks`, `.formSubmitButton` etc., so adding `.description` with all its styling in one place probably makes sense, too.

I wouldn't stress too much for this change, but we should be aware of the friction and start pushing on unifying the UI componentry and approach to things like this more and more with the product owner and other folks in the wider team so we get consistency, usability and a11y, and not increase technical debt.

Reply:
My intention on having descriptive class names in favor of functional ones is, that in a perfect world - and I know that we are far away from that - we would have a clean HTML structure and descriptive class names for all (relevant) elements.
That would allow additional themes to fully adjust the layout. As themes can only provide CSS and can't overwrite all parts of the HTML easily, classes like `uppercase` won't be helpful anymore. There might afterwards be places where you still want the uppercase but others where you want to remove it, which would then only be possible by using complex rules to overwrite the existing ones. Using element styling and descriptive class names would be a lot more flexible.

Reply:
That's a valid point that I forgot to factor in — theming.

Reply:
@sgiehl Updated as suggested, please check.